src/providers/ethereum/web3InjectedWallet.ts

```
@@ -0,0 +1,102 @@
+            this.address = account[0] as HexPrefixedString
+            resolve(true)
+          }
+        } else {
+          resolve(false)
+        }
+
+      } catch (err) {
+        reject(err)
+      }
+
+    })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)